### DISCUSSION TITLE
No scale down node event

### DISCUSSION THREAD

**Discussion by xpicio** [argoproj/argo-cd#8835] [Q&A]
Hello, on a GKE cluster 1.21.9-gke.1002 with ArgoCD 2.2.5 i'm receiving the following event:

```
{
  "insertId": "c6db8e89-f210-449e-a4df-0ea22fe60c76@a1",
  "jsonPayload": {
    "noDecisionStatus": {
      "noScaleDown": {
        "nodes": [
          {
            "reason": {
              "parameters": [
                "argocd-repo-server-8465d84869-5j2pr"
              ],
              "messageId": "no.scale.down.node.pod.has.local.storage"
            },
            "node": {
              "cpuRatio": 46,
              "memRatio": 47,
              "name": "gke-stg-europe-west1-default-pool-01-9d0e543b-mdm7",
              "mig": {
                "zone": "europe-west1-b",
                "nodepool": "default-pool-01",
                "name": "gke-stg-europe-west1-default-pool-01-9d0e543b-grp"
              }
            }
          }
        ],
        "nodesTotalCount": 1
      },
      "measureTime": "1647849784"
    }
  },
  "resource": {
    "type": "k8s_cluster",
    "labels": {
      "project_id": "project-id-stg",
      "location": "europe-west1-b",
      "cluster_name": "stg-europe-west1"
    }
  },
  "timestamp": "2022-03-21T08:03:04.774904402Z",
  "logName": "projects/project-id-stg/logs/container.googleapis.com%2Fcluster-autoscaler-visibility",
  "receiveTimestamp": "2022-03-21T08:03:05.050628197Z"
}
```

Checking the volumes used by the pod it's seems that the data is temporary or for caching purpose, so it it safe add the following annotation `"cluster-autoscaler.kubernetes.io/safe-to-evict": "true"` ? Maybe could be add to the official setup ?

**Comment by jsoref** [comment]
How did you install ArgoCD?

We're using argo-helm, whose configuration knobs are defined here:
https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd#argo-cd-configs

by toggling all 7 `pdb` knobs, we're able to get PodDisruptionBudgets managed by ArgoCD (ArgoCD manages ArgoCD's deployment for us) which should`*` result in this problem not persisting.

`*` We just deployed this now, and haven't had enough time to determine whether it will cover this case. the documentation for the autoscaler is somewhat ambiguous as to whether a PDB for a pod is enough to allow it to be evicted even if it has local storage.
